dotnet/src/Functions/Functions.Grpc/Extensions/KernelGrpcExtensions.cs

```diff
@@ -195,4 +196,47 @@ async Task<SKContext> ExecuteAsync(SKContext context)
     }
 
     #endregion
+
+    #region obsolete
+
+    [Obsolete("Methods and classes which includes Skill in the name have been renamed to use Plugin. Use Kernel.ImportGrpcFunctionsFromDirectory instead. This will be removed in a future release.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CS1591
+    public static IDictionary<string, ISKFunction> ImportGrpcSkillFromDirectory(
+        this IKernel kernel,
+        string parentDirectory,
+        string skillDirectoryName,
+        HttpClient? httpClient = null)
+    {
+        return kernel.ImportGrpcFunctionsFromDirectory(parentDirectory, skillDirectoryName, httpClient);
+    }
+#pragma warning restore CS1591
+
+    [Obsolete("Methods and classes which includes Skill in the name have been renamed to use Plugin. Use Kernel.ImportGrpcFunctionsFromFile instead. This will be removed in a future release.")]
+    [EditorBrowsable(EditorBrowsableState.Never)]
+#pragma warning disable CS1591
```

Comment:
nit: use `SuppressMessage` attribute on these methods

Reply:
We've actually split usage quite a bit in the repo. I kind of prefer the pragma warnings just because it doesn't require a namespace using and can apply to files or specific lines more easily. Are there other considerations here on why we might want to use SuppressMessage instead of the pragma markers?